scripts/pathfinders_validation.py

```diff
@@ -0,0 +1,559 @@
+    INTERVENTION_THEMES = [
+        "Enhancing sub-regional and regional connectivity",
+        "Unlocking and enabling industrial, commercial, and residential development",
+        "Unlocking and enabling industrial and commercial development",
+        "Strengthening the visitor and local service economy",
+        "Improving the quality of life of residents",
+        "Employment & Education",
+    ]
+    RISK_CATEGORIES = [
+        "Strategy risks",
+        "Programme or project delivery risks",
+        "People risks",
+        "Operational process or control risks",
+        "Resilience risks",
+        "Governance risks",
+        "System or IT infrastructure risks",
+        "Security risks",
+        "Financial risks",
+        "Commercial or procurement risks",
+        "Lega, regulatory or compliance risks",
+        "Arm’s length body risks",
+        "Local government risks",
+        "Political sensitivity",
+        "Reputational Risk",
+        "Resource & Expertise",
+        "Slippage",
+        "Planning Permission / other consents",
+        "Procurement / contracting",
+        "Delivery Partner",
+        "Financial flexibility",
+        "Subsidy Control",
+        "Other",
+    ]
+    RISK_SCORES = ["1 - very low", "2 - low", "3 - medium", "4 - high", "5 - very high"]
+    FORECAST_ACTUAL = ["Forecast", "Actual"]
+    SPEND_TYPE = [
+        "How much of your forecast is contractually committed?",
+        "Freedom and flexibilities spend",
+        "Total DLUHC spend (incl F&F)",
+        "Secured Match Funding Spend",
+        "Unsecured Match Funding ",
+        "Total Match ",
+    ]
+    REPORTING_PERIOD = [
+        "Q1 Apr - Jun 23/24",
+        "Q2 Jul - Sep 23/24",
+        "Q3 Oct - Dec 23/24",
+        "Q4 Jan - Mar 23/24",
+        "Q1 Apr - Jun 24/25",
+        "Q2 Jul - Sep 24/25",
+        "Q3 Oct - Dec 24/25",
+        "Q4 Jan - Mar 24/25",
+        "Q1 Apr - Jun 25/26",
+        "Q2 Jul - Sep 25/26",
+        "Q3 Oct - Dec 25/26",
+        "Q4 Jan - Mar 25/26",
+        "Apr 26 onwards",
+    ]
+
+
+class PFErrors:
+    is_float = (
+        "You entered text instead of a number. Remove any units of measurement and only use numbers, for example, 9."
+    )
+    is_datetime = "You entered text instead of a date."
+    isin = (
+        "You’ve entered your own content, instead of selecting from the dropdown list provided. Select an "
+        "option from the dropdown list."
+    )
+    lt_100_words = "Please enter a maximum of 100 words."
+
+
+PF_CONFIG = {
+    "Financial Completion Date": {
+        "extract": {"id_tag": "PF-TABLE-ID-01", "worksheet_name": "Admin"},
+        "process": {
+            "ignored_non_header_rows": [0],
+        },
+        "validate": {
+            "columns": {
+                "Pathfinder Financial Completion Date": pa.Column(
+                    datetime, pa.Check.is_datetime(error=PFErrors.is_datetime)
```

Comment:
I've looked into this and it doesn't look like you can unfortunately. One other option would be to not register them as check methods. This wouldn't really have any downsides as we don't use the two main advantages of registering check functions anyway (particularly as I've found the data synthesis extremely slow after some testing), see...

> However, there are two main disadvantages of schemas with inline custom checks:
>     1. they are not serializable with the IO interface.
>     2. you can’t use them to synthesize data because the checks are not associated with a hypothesis strategy.

I did originally implement the checks like this...
```
class Check:

    def is_int(error="Default error message"):
        return pa.Check(_is_int_check, error=error)
```
but then opted for registering as built-in checks in the end. I think for now I'm happy as is and it can be revisited in later work if we want to change it.